src/components/common/modal/bulk-edit-modal.tsx

```diff
@@ -26,6 +27,8 @@ export const BulkEditModal = ({ modal }: IProps) => {
             items: dropdownItemsString,
         },
     });
+
+    const dropdownItemsErrors = methods.formState.errors;
```

Comment:
the validity should be assessed right from the start, i.e. if the options are invalid, the submit button should be disabled without any user intervention

Reply:
perform validation manually to get the form validity yet not show the error message 
```
try {
        BULK_EDIT_SCHEMA.validateSync({ items: methods.watch("items") });
} catch (e) {}
```